.gitignore

```diff
@@ -2,3 +2,4 @@ coverage/
 node_modules/
 build/
 storybook-static/
+.idea/
```

Comment:
We don't add IDE-specific paths to the gitignore. Add them to your local ignore list

Reply:
Thanks,I didn't knew it